examples/kubernetes/vtctld-service.yaml

```diff
@@ -14,4 +14,4 @@ spec:
   selector:
     component: vtctld
     app: vitess
-
+  type: LoadBalancer
```

Comment:
I don't think this hurts the use case where we do not expose vtctld externally, which is why I didn't go through the effort of making this conditional on TEST_MODE.